### ISSUE TITLE
algorithms should be set (readme incomplete?)

### ISSUE THREAD

**Issue by Morriz** [auth0/express-jwt#237]
### Description

When following the readme I get the following error:
```
algorithms should be set
```

### Reproduction

Don't provide the algorithms in the jwt object creation.

**Comment by mcastany** [comment]
Closed https://github.com/auth0/express-jwt/pull/234

**Comment by blackdynamo** [comment]
Why was this closed?

Based on this commit, it requires you to pass in `options.algorithms` but the code doesn't even use it.
https://github.com/auth0/express-jwt/commit/304a1c5968aed7c4c520035426fc09142156669d

**Comment by blackdynamo** [comment]
For anyone else who might get bit by this, you just need to provide a payload like this:

The options object gets passed on to the jsonwebtoken module.

```
const jwtCheck = jwt({
  algorithms: ["HS256"], // and/or other algorithms you are using
  secret: "the secret",
  ...otherOptions // issuer, audience, etc.
});
```

**Comment by Zenkylo** [comment]
No idea why this isn't in the readme. Can confirm above algorithm**s** option works.

**Comment by robations** [comment]
The doc change was made _after_ 6.0.0 was published so the master here and 6.0.0 on npmjs.com are out of step.

https://www.npmjs.com/package/express-jwt

It might save some confusion if a 6.0.1 version was pushed to resolve this inconsistency — usually the npm page is what I see first if I'm researching a package.

**Comment by justinmchase** [comment]
Probably a version 6.0.1 should be published just to get the doc changes updated.
